README.md

```diff
@@ -21,5 +21,8 @@ The Test Harness is a CLI that you need to install:
 - `pip install -r requirements-runners.txt` to install the Test Runners
 - `pip install .` to install the Test Harness CLI
 
+Create a local 'logs' directory (the repository doesn't track or commit this...):
```

Comment:
Can we add a `.gitkeep` file to the logs folder so it is tracked? I would rather have that than make the user create the directory themselves.